client-misk/build.gradle

```diff
@@ -2,14 +2,6 @@ apply plugin: 'com.squareup.wire'
 apply plugin: 'kotlin'
```

Comment:
For the `src/main/proto` path not existing compile error, you'll need to remove the wire plugin from `client-misk`